sgeop/tests/test_simplify.py

```diff
@@ -87,6 +87,7 @@ def test_simplify_network_full_fua(aoi, tol, known_length):
         geopandas.read_parquet(full_fua_data / aoi / "original.parquet")
     )
     observed_length = observed.geometry.length.sum()
+    assert "highway" in observed.columns
```

Comment:
@jGaboardi I am wondering if we want to update the known parquets to contain all the other columns and test their equality or if this is enough.

Reply:
I think this is enough for now, maybe with #135 we also can reset the indices so the parquet also have `['u', 'v', 'key']`. That's probably enough, right?